PcBdsPkg/Library/MsBootOptionsLib/MsBootOptionsLib.c

```diff
@@ -423,8 +421,8 @@ RegisterFvBootOption (
     BootOptions = EfiBootManagerGetLoadOptions (&BootOptionCount, LoadOptionTypeBoot);
 
     if (BootOptions == NULL) {
-      ASSERT (BootOptions != NULL);
-      return MAX_UINTN;
+      // ASSERT (BootOptions != NULL);

```

Comment:
Remove the code instead of commenting if not needed.

Reply:
Fixed.